content/kubermatic/master/tutorials_howtos/external_clusters/eks/_index.en.md

```diff
@@ -5,7 +5,54 @@ weight = 7
 
 +++
 
+## Add EKS Cluster
 
+You can add an existing Kubernetes cluster and then manage it using KKP.
+From the `Clusters` page, click `External Clusters`. Click the `Add External Cluster` button and pick `Elastic Kubernetes Engine` provider.
 
+![Add External Cluster](/img/kubermatic/master/tutorials/external_clusters/add_external_cluster.png "Add External Cluster")
 
+Select preset with valid credentials or enter EKS `Access Key ID`, `Secret Access Key` , and `Region` to connect to the provider.
+
+![EKS credentials](/img/kubermatic/master/tutorials/external_clusters/eks_credentials.png "EKS credentials")
+
+You should see the list of all available clusters in the region specified. Select the one and click the `Import Cluster` button. Clusters can be imported only once in a single project. The same cluster can be imported for the other projects.
+
+![Select EKS cluster](/img/kubermatic/master/tutorials/external_clusters/select_eks_cluster.png "Select EKS cluster")
+
+## Cluster Details Page
+
+After the cluster is added, you can view the cluster details.
+The KKP controller retrieves the cluster kubeconfig to display all necessary information. A healthy cluster shows `Running` state.
```

Comment:
Looks like some of the information here is duplicated on other pages. It's better that way than having too little but that's a good thing to improve later. Not required now as the release is really close.